pages/common/pssh.md

```diff
@@ -0,0 +1,27 @@
+# pssh
+
+> Parallel ssh program.
+
+- Connect to host1 and host2 to print the date on each server inline:
+
+`pssh -i -H "{{host1}} {{host2}}" date`
+
+- Run a command and save the output to separate files:
+
+`pssh -H host1 -H host2 -o {{path/to/output_dir}} date`
+
+- Run a command on each host specified in a new-line separated host files:
+
+`pssh -i -h {{path/to/hosts_file}} date`
+
+- Run a commond as root, prompting for root's password:
+
+`pssh -i -h {{path/to/hosts_file}} -A -l root date`
+
+- Run a command with extra SSH arguments:
+
+`pssh -i -h {{path/to/hosts_file}} -x "-O StrictHostKeyChecking=no" date`
```

Comment:
Is that really a good example additional option? I'm not sure we should use that one as an example for security reasons :P

Also, it should be surrounded in `{{}}` because you can edit it.